topdown/internal/jwx/buffer/buffer_test.go

```diff
@@ -0,0 +1,89 @@
+package buffer
```

Comment:
I'm noticing that none of these files have copyright headers on them. Other spots in OPA have explicit ones... should we add some? Were there any that require new attributions (significant rewrites from the original jwx project?)

For the original MIT license we are supposed to keep the original license and attribution around somewhere right? To cover ourselves maybe just drop a copy of the original license in the `topdown/internal/jwx` directory? (/me is not a lawyer).

Reply:
Good suggestion. Added original license. I've done extensive rewrites in some parts but I do not _think_ new attributions are needed.